static-site/src/components/Showcase/index.tsx

```diff
@@ -112,65 +110,13 @@ const PreviewOverlay = styled.div`
 
 const CardsContainer = styled.div`
   /* background-color: #ffeab0; */
-  display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(${cardWidthHeight}px, max-content));
-  grid-gap: 1%;
+  display: inline-flex;
+  flex-wrap: wrap;
+  gap: 10px;
```

Comment:
I would use a smaller (or no) row-gap to better match the previous (grid) layout where we could mostly read the titles of the collapsed row. An alternative solution would be to keep the gap as-is and increase the space allocated to the UI when collapsed

<img width="1178" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/229153af-1449-43f1-8fd6-07d02f90dbe1">

Reply:
Thanks for pointing this out. I realized this was a side effect of using a relative value for `gap`. Fixed in dcd8a92829f110d5be32280cb857c2d27ec6d1b6.

This also prompted me to add some other CSS grid fixes to this PR: 883998a33a2a0f43e6d3eac4b82976ccb913c017, f8388ea0f5bda3ca08a6c3e9a21030d56ccd29f2